Project-Hotpot/Project-Hotpot/Managers/GenreQueueManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  GenreQueueManager.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/26/22.
+//
+
+import Foundation
+
+class GenreQueueManager: NSObject {
```

Comment:
You could just call it GenreManager. Generally we should avoid putting implementation details in class names.